.github/workflows/buildReleaseAndPublishWindows.yml

```diff
@@ -0,0 +1,80 @@
+name: Build Release and Publish windows wheels
+
+on:
+  workflow_dispatch:
```

Comment:
can you add a `pull_request:` temporarily here? This way we can see how long it takes and if it works as intended :)

Reply:
Thanks for the review @saienduri. I've made the suggested edit but I don't have permission to run the workflow. Can you trigger it please when you get a chance? Thanks!

Reply:
Yeah, triggered and started a run

Reply:
Thanks @saienduri, looks like it failed due to permissions probably because this is triggered via the PR. The action does run fine in my repo https://github.com/sahas3/torch-mlir-release/actions/runs/11487806632. To debug further, I made a temporary change to elevate all permissions but I can't trigger the workflow myself. What will be best way to debug this further in this repo -- any suggestions?

Reply:
I recently got write access to LLVM but looks like that doesn't cover this repository, so I still cannot trigger the workflow in here to debug the issue further. Is it possible to get write access for this repository? If not, @saienduri  do you have any suggestions for debugging the permission issue as the change runs fine in my forked repo. Thanks!

Reply:
Hello @saienduri, @sjain-stanford  -- checking in to see if I can get write access in this repo (temporarily is fine) to further debug this issue as I cannot repro it on my own forked repo. Thanks!

Reply:
Sorry for the delayed response, @sahas3. I've triggered the runs again, let's see where we get:
https://github.com/llvm/torch-mlir-release/actions/runs/11489688392

If this fails and you want to test things out on a branch on this repo, @stellaraccident  would it be alright to provide temp write access to @sahas3? I'd be OK but I don't fully understand the risks of elevating permissions for `GITHUB_TOKEN` as done here:
```
     # Elevates permissions for `GITHUB_TOKEN`'s content scope (to allow this action to make a release)
-    permissions:
-      contents: write
+    permissions: write-all
```

Reply:
Sorry, missed this. I think the point of adding the `pull_request` and testing here was to see if we would get a successful build. The rest should work on TOM (just uploading wheels), so I think it should be fine to merge and see what happens. What do y'all think?

Reply:
Yeah, I am puzzled by the failure. Everything I've found so far indicates that 
```
 permissions:
      contents: write
```
should be sufficient. `permissions: write-all` is a heavy hammer that I tried just to figure out  if any other permission is required but that also failed. I wonder if this permission issue has to do something with the action being triggered from the PR. 

The code for updating wheels is same as the Linux one, so I think (well I am biased in this regards :D) it's worth it to merge the change and see if that part runs fine when the action is triggered normally from the repo.

Reply:
Yup, I agree. Let's try it out. worst case we can revert :)

Reply:
I don't think the GITHUB_TOKEN itself is accessible on forks. You can revert your permission change and we can land :)

Reply:
Thanks @saienduri. Reverted the temporary changes.

Reply:
Thanks @saienduri, can you please merge this PR as well whenever you get a chance.